docs-src/python/how-to-spawn-a-child-workflow-execution-in-python.md

```diff
@@ -0,0 +1,21 @@
+---
+id: how-to-spawn-a-child-workflow-execution-in-python
+title: How to spawn a Child Workflow Execution in Python
+sidebar_label: Child Workflow Execution
+tags:
+  - developer-guide
+  - python
+---
+
+To spawn a Child Workflow Execution in Python, use the [`execute_child_workflow()`](https://python.temporal.io/temporalio.workflow.html#execute_child_workflow) function. `execute_child_workflow()` starts the Child Workflow and waits for completion.
+
+```python
+await workflow.execute_child_workflow(MyWorkflow.run, "my child arg", "my-child-id")
```

Comment:
```suggestion
await workflow.execute_child_workflow(MyWorkflow.run, "my child arg", id="my-child-id")
```

Things have changed since. ID is optional. https://github.com/temporalio/documentation/issues/1458#issuecomment-1234537286 is no longer accurate.